src/mame/mr/nl_dribling.cpp

```diff
@@ -0,0 +1,524 @@
+	NET_MODEL("BC309 PNP(IS=1.0366E-15 ISE=1.6295E-14 ISC=9.4971E-17 XTI=3 BF=80 BR=16.116 IKF=0.47497 IKR=0.012081 XTB=0 VAF=8.486 VAR=9.5149 VJE=0.84456 VJC=1.0282 RE=1.9597 RC=1.1393 RB=3.2133 CJE=2.0636E-14 CJC=2.6904E-12 XCJC=0.075977 FC=0.54298 NF=1.03136 NR=1.2907 NE=1.3702 NC=1.038 MJE=0.68352 MJC=0.5401 TF=5.4303E-11 TR=3.4233E-10 EG=1.11 IRB=0.00046855 RBM=2.0822 XTF=0.27447 VTF=0.19311 ITF=0.0030573 PTF=0 CJS=0 VJS=0.75 MJS=0)")
+
+	SOLVER(Solver, 1000)
+	PARAM(Solver.DYNAMIC_TS, 1)
+	PARAM(Solver.DYNAMIC_MIN_TIMESTEP, 1e-5)
+
+	CLOCK(clk, 40000) // 40KHz
+
+	LOCAL_SOURCE(LM339)
+	LOCAL_SOURCE(calcio)
+	LOCAL_SOURCE(stop_palla_parata)
+
+	ANALOG_INPUT(I_V5, 5)
+	ANALOG_INPUT(I_V6, 6)
+	ANALOG_INPUT(I_V12, 12)
+
+	TTL_7414_DIP(IC_L9) // SHARED INVERTER, FISCHIO 1-2, TOS 3-9
+	NC_PIN(NC)
+	NET_C(IC_L9.10, GND.Q)
+	NET_C(IC_L9.12, GND.Q)
+
+	
+	
+	// TOS Section
+	
+	TTL_INPUT(I_PB0, 0)
+	TTL_INPUT(I_PB1, 0)
+	TTL_INPUT(I_PB2, 0)
+	TTL_INPUT(I_PB3, 0)
+	TTL_INPUT(I_PB4, 0)
+	TTL_INPUT(I_PB5, 0)
+	TTL_INPUT(I_PB6, 0)
+	TTL_INPUT(I_PB7, 0)
+	NET_C(GND, I_PB0.GND, I_PB1.GND, I_PB2.GND, I_PB3.GND, I_PB4.GND, I_PB5.GND, I_PB6.GND, I_PB7.GND, clk.GND)
+	NET_C(I_V5, I_PB0.VCC, I_PB1.VCC, I_PB2.VCC, I_PB3.VCC, I_PB4.VCC, I_PB5.VCC, I_PB6.VCC, I_PB7.VCC, clk.VCC)
+	CAP(C7_1, CAP_P(330))
+	CAP(C7_2, CAP_P(330))
+
+	TTL_74161_DIP(IC_D7)  // 4 bit counter
+	TTL_74161_DIP(IC_E7)  // same
+	TTL_74LS107_DIP(IC_C7)  // dual JK flip flop
+
+	NET_C(GND, IC_C7.7, IC_D7.8, IC_E7.8, C7_1.2)
+	NET_C(I_V5, IC_C7.14, IC_D7.16, IC_E7.16)
+
+	NET_C(I_PB0, IC_D7.3)
+	NET_C(I_PB1, IC_D7.4)
+	NET_C(I_PB2, IC_D7.5)
+	NET_C(I_PB3, IC_D7.6)
+	NET_C(I_PB4, IC_E7.3)
+	NET_C(I_PB5, IC_E7.4)
+	NET_C(I_PB6, IC_E7.5)
+	NET_C(I_PB7, IC_E7.6)
+
+	NET_C(IC_E7.1, IC_D7.1, I_V5)  // CLEAR FLAG 
+	NET_C(IC_D7.7, IC_D7.10, I_V5)
+	NET_C(IC_E7.9, IC_D7.9, C7_1.1, IC_L9.4)
+	NET_C(IC_D7.15, IC_E7.7, IC_E7.10)
+	NET_C(IC_E7.15, IC_L9.3)
+	NET_C(IC_L9.6, IC_E7.2, IC_D7.2)
+	NET_C(IC_L9.5, C7_2.1, IC_C7.9, IC_L9.8)
+	NET_C(GND, C7_2.2)
+	NET_C(IC_L9.9, clk)
+
+	NET_C(IC_C7.10, I_V5)
+	NET_C(IC_C7.1, IC_C7.13, IC_C7.4, IC_C7.12, GND.Q)
+	NET_C(IC_C7.8, IC_E7.15) // FLIP-FLOP
+	NET_C(IC_C7.11, IC_E7.15)
+
+#ifdef FILTER_TOS
+	// ADDITIONAL CIRCUITRY TO FILTER 40Khz output from TOS;
+	// AND (flip-flop, NAND(all input pins))
+	TTL_7430_DIP(NAND)
+	NET_C(I_PB0, NAND.1)
+	NET_C(I_PB1, NAND.2)
+	NET_C(I_PB2, NAND.3)
+	NET_C(I_PB3, NAND.4)
+	NET_C(I_PB4, NAND.5)
+	NET_C(I_PB5, NAND.6)
+	NET_C(I_PB6, NAND.11)
+	NET_C(I_PB7, NAND.12)
+	NET_C(NAND.7, GND)
+	NET_C(NAND.14, I_V5)
+
+	TTL_7408_DIP(AND)
+	NET_C(NAND.8, AND.1)
+	NET_C(IC_C7.5, AND.2)
+	NET_C(GND.Q, AND.4, AND.5, AND.9, AND.10, AND.12, AND.13)
+	NET_C(GND, AND.7)
+	NET_C(I_V5, AND.14)
+	ALIAS(TOS, AND.3)
+#else
+	// Use original flip-flop output as from schematics.
+	ALIAS(TOS, IC_C7.5)
+#endif
```

Comment:
It’s pretty common for these analog sound systems to produce high-frequency idle sound that won’t make it through the audio outputs (e.g. listen to Cheeky Mouse in MAME, `cheekyms`).  Is it really a good idea to add fake circuitry like this?

Reply:
I would argue that such bandpass filtering, if desired, should be handled outside the netlist subsystem (e.g. in the audio output pipeline, after any/all effects have been processed), and have an on/off setting toggle. It definitely does not belong in the netlist, unless it was present on original hardware.

Reply:
Yeah, I've tried that leaving the sound on but it's unbearable since the output pipeline doesn't help here. The square wave of the 40Khz clock leaves too many harmonics out.
Consider also that the schematic uses a TDA2003 for final amplification (and pass-band I guess) that's not available. 
If there are better alternatives, I more than willing to pursue them.